src/components/barber.tsx

```diff
@@ -0,0 +1,15 @@
+import { Barber } from "../domains/barber/types";
+export function Barb({ name, services }: Barber) {
+  return (
+    <>
+      <h2>{name}</h2>
+      <p>Hairdresser-colorist</p>
+      <ul>
+        Services
+        {services.map((service, index) => (
+          <li key={index}>{service}</li>
```

Comment:
Якщо є `id`, то краще використовувати його як ключ.
```suggestion
        {services.map((service) => (
          <li key={service.id}>{service}</li>
```